src/elements/fields/SignatureField/components/SignatureModal.tsx

```diff
@@ -0,0 +1,375 @@
+    onEnd(signatureFile);
+    setShow(false);
+    resetState();
+  };
+
+  if (!show) {
+    return null;
+  }
+
+  return (
+    <div
+      css={{
+        position: 'fixed',
+        display: 'flex',
+        top: 0,
+        left: 0,
+        width: '100vw',
+        height: '100vh',
+        background: 'rgba(0, 0, 0, 0.2)',
+        zIndex: 1000,
+        alignItems: 'center',
+        justifyContent: 'center',
+        fontSize: '16px'
+      }}
+    >
+      <div
+        onClick={() => setShow(false)}
+        css={{
+          position: 'fixed',
+          top: 0,
+          left: 0,
+          width: '100%',
+          height: '100%'
+        }}
+      />
+      <div
+        className='feathery-modal'
+        css={{
+          position: 'relative',
+          backgroundColor: '#fff',
+          borderRadius: '14px',
+          width: '100%',
+          maxWidth: '600px'
+        }}
+      >
+        <div
+          css={{
+            position: 'relative',
+            display: 'flex',
+            padding: '20px 20px',
+            borderBottom: '1px solid #e9e9e9'
+          }}
+        >
+          <h3 css={{ padding: 0, margin: 0, flex: '1' }}>Add your signature</h3>
+          <CloseIcon
+            onClick={() => setShow(false)}
+            css={{ '&:hover': { cursor: 'pointer' } }}
+          />
+        </div>
+        <div
+          css={{
+            position: 'relative',
+            padding: '30px 20px',
+            '& h3': {
+              fontSize: '1em',
+              margin: 0,
+              padding: 0
+            }
+          }}
+        >
+          {!drawSignature && (
+            <>
+              <div
+                css={{
+                  display: 'flex',
+                  gap: '15px',
+                  flexDirection: 'column',
+                  paddingBottom: '30px'
+                }}
+              >
+                <h3>Type your signature</h3>
+                <input
+                  onChange={(e) => {
+                    setFullName(e.target.value);
+                    setLoading(true);
+                    debounceGenerateSignature();
+                  }}
+                  placeholder='Your full name'
+                  css={{
+                    padding: '8px 10px',
+                    borderRadius: '4px',
+                    border: '1px solid #e9e9e9',
+                    '&:focus,&:focus-visible': {
+                      border: '1px solid #5e5e5e',
+                      outline: 'none'
+                    }
+                  }}
+                />
+                <div
+                  css={{
+                    position: 'relative',
+                    width: '100%',
+                    height: '100px',
+                    backgroundColor: '#f6f6f6',
+                    display: 'flex',
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                    fontSize: '2em',
+                    fontFamily: 'cursive'
```

Comment:
Overflow hidden.  Edge case: Long names can overflow the box.  
![image](https://github.com/feathery-org/feathery-react/assets/16427572/59712c26-9342-47a6-b563-bb4144b094a9)